js/addCardForm/index.js

```diff
@@ -25,3 +25,28 @@ form.onsubmit = (event) => {
 
   form.reset();
 };
+
+//character limit functionality
+const textareas = document.querySelectorAll('[data-js="textarea"]');
+const outputCharacters = document.querySelectorAll(
+  '[data-js="output-characters"]'
+);
+
+// Loop through each textarea element
+textareas.forEach((textarea, i) => {
```

Comment:
I recommend using speaking names for our arguments, such as index in this case:
```suggestion
textareas.forEach((textarea, index) => {
```

That makes it a lot clearer as to what it actually stands for